node/src/Pos/Communication/Tx.hs

```diff
@@ -49,10 +48,10 @@ type TxMode ssc m
       , TxCreateMode m
       )
 
-submitAndSaveTx
+submitAndSave
```

Comment:
This is an extra change I made in CSM-256 and now name of this function was confusing (`submitTx` called `submitAndSaveTx`)